src/TextInputWithTokens.tsx

```diff
@@ -246,7 +252,9 @@ function TextInputWithTokensInnerComponent<TokenComponentType extends AnyReactCo
       width={widthProp}
       minWidth={minWidthProp}
       maxWidth={maxWidthProp}
-      variant={variantProp}
+      size={inputSize}
+      validationStatus={validationStatus}
```

Comment:
can we add an `aria-invalid` attribute here too if `validationStatus === 'error'`?

Reply:
Yes, we should.